src/mailbox/ui/App.js

```diff
@@ -70,7 +76,17 @@ module.exports = React.createClass({
 
 		// Tell the app about our unread count
 		const unread = store.totalUnreadCount()
-		app.dock.setBadge(unread ? unread.toString() : '')
+		if(process.platform === 'darwin') {
+			app.dock.setBadge(unread ? unread.toString() : '')
+		} else {
+			const unreadText = unread ? unread + ' unread mail' : 'No unread mail';
+			const appIcon = appTray || (appTray = new Tray(path.resolve('icons/app.png')));
```

Comment:
Need to save appTray to a module global to avoid recreation on each dom refresh. Weird...